dir.props

```diff
@@ -16,18 +16,14 @@
 
   <!-- Common nuget properties -->
   <PropertyGroup>
-    <NuGetToolPath>$(ToolsDir)NuGet.exe</NuGetToolPath>
+    <NuGetToolPath>$(PackagesDir)NuGet.exe</NuGetToolPath>
```

Comment:
ToolsDir points inside of our extracted buildtools package and I didn't want put files like this into that package because I wanted leave the ToolsDir untouched. We could have choosen another location besides PackagesDir but I think it makes the most sense, given what it is used for.